### DISCUSSION TITLE
[急][委托]我家邦布的斗魂完全熄灭了

### DISCUSSION THREAD

**Discussion by TianxingOO** [share121/inter-knot#2124] [其他]
大家好!我是「邦布力士联赛」的一名教练，现在向各位发出特训助教的招募邀请!
为了将邦布的肉体与精神都锤炼至更高更强的境界，我制定了周密、科学而且还很热血的训练计划! 
我准备把选手带到空洞，与空洞中最强壮的力士们角斗!怎么样，是不是光听听就燃起来了? 
为了执行这个计划，特此公开聘请靠谱的绳匠作为助教，协助选手完成训练计划! 
*由于先前未经严格审核的低水平助教贸然接下委托，导致训练计划受挫，请欲接取委托的各位绳匠向我发送包含过往主要委托成果和项目履历的简历一份!

**Comment by TianxingOO** [comment]
![cd8d3c82b9b4fd74cff7779ce5bad03](https://github.com/user-attachments/assets/934a6964-2c6c-4955-93c3-bad4710ab9e6)

**Comment by wesdyuhj** [comment]
这个委托我接了。为您附上阿哈的简历。
